packages/wonder-blocks-form/components/choice-field.md

```diff
@@ -0,0 +1,68 @@
+ChoiceField is to be used independently. If you wish to use choices as part of a
```

Comment:
Would be good to emphasize that the radio is probably useless.

🛑 Although, I think it would actually be better to just make it so that ChoiceField only supports checkbox. We could add in radio later, if need be, but I just worry about people getting really confused and using the wrong thing!